deployer/src/config_manager.py

```diff
@@ -1,12 +1,10 @@
 import algoliasearch
-import json
 from os import environ
 
 from . import algolia_helper
 from . import snippeter
 from . import emails
 from . import helpers
-from .dict_differ import DictDiffer
```

Comment:
(just as a side note: you could have deleted dict_diff.py in the same commit; the rationale being that otherwise, after that commit, dict_diff.py is still around in the repository, while not being leveraged; it's dead code)